test/image-server.etags.json

```diff
@@ -68,5 +68,7 @@
 	"http://localhost:13337/Portrait_5.jpg/:/rs=w:320?cache=false": "3340085419",
 	"http://localhost:13337/Portrait_6.jpg/:/rs=w:320?cache=false": "2612685063",
 	"http://localhost:13337/Portrait_7.jpg/:/rs=w:320?cache=false": "783772049",
-	"http://localhost:13337/Portrait_8.jpg/:/rs=w:320?cache=false": "4137304836"
+	"http://localhost:13337/Portrait_8.jpg/:/rs=w:320?cache=false": "4137304836",
+	"http://localhost:13337/UP_steam_loco.jpg/:/cr=w:50%25,h:50%25,at:100,al:200/fm=f:jpeg?cache=false": "2536073760",
+	"http://localhost:13337/UP_steam_loco.jpg/:/cr=w:50%25,h:50%25,ay:100,ax:200/fm=f:jpeg?cache=false": "2536073760"
```

Comment:
You're correct in that it was producing the same result as anchoring at `tl` and was thus not a good test, but it's not because the code is broken - it's because the test was not a good one. If you specify an absolute anchor position at (10, 10) say, and you're trying to crop a region that is 100x100, it will automatically adjust your anchor position so the full region can be cropped - so it ends up being equivalent to anchoring at `tl`.